src/Storage/Masa.BuildingBlocks.Storage.ObjectStorage/Response/ResponseBase.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Masa.BuildingBlocks.Storage.ObjectStorage.Response;
+public class ResponseBase
```

Comment:
The reason for not using Exception is to worry that users in some scenarios need to get the status, and other solutions are used for unsuccessful ones, so ResponseBase is used instead of throwing exceptions directly